revised gen database for CEJA

ssattler opened the pull request:
Revised the EIA-NEMS database to reflect CEJA

From James Gignac:
https://ucsusa-my.sharepoint.com/:x:/r/personal/jgignac_ucs_org/_layouts/15/Doc.aspx?sourcedoc=%7B94B54D83-D789-4D97-829B-B5F5547BF70A%7D&file=CEJA%20plant%20retirements%20for%20ReEDS.xlsx&action=default&mobileredirect=true

The following plants are already retired:
1515 South Caron Road – Retired March 2022
Freedom Power Project – Retired May 2024
Grand Tower Energy C – Retired Sept. 2023
Southeast Chicago En – Retired March 2024

Morris Cogeneration – Set retirement for 2045

ssattler committed:
revised gen database for CEJA

Revised the EIA-NEMS database to reflect CEJA

samgdotson reviewed (changes requested):
Thanks, Sandra. 

Would you mind adding the updated state-cap file to this pull request?

ssattler commented:
This run is with just the LA cap, correct?

ssattler committed:
revised LA emissions cap

state_cap file revised for LA total emissions.

samgdotson commented:
I think we had a co2 cap for the other states because I originally built this on top of the `state-cap` version. The main difference was the RPS for LA vs a CES.

ssattler committed:
revised state_cap for WI and MI

revised the state_cap file for LA, WI and MI for total state emissions to calculate the state carbon cap.
